weblate/api/serializers.py

```diff
@@ -85,13 +85,25 @@ class Meta:
         model = Language
         fields = ("code", "name", "direction", "web_url", "url")
         extra_kwargs = {
-            "url": {"view_name": "api:language-detail", "lookup_field": "code"}
+            "url": {"view_name": "api:language-detail", "lookup_field": "code"},
+            "code": {"validators": []},
         }
 
+    def validate_code(self, value):
+        check_query = Language.objects.filter(code=value)
+        if check_query.exists() and not (
+            isinstance(self.parent, ProjectSerializer)
+            and self.field_name == "source_language"
+        ):
+            raise serializers.ValidationError(
+                "Language with this Language code already exists."
```

Comment:
This ValidationError is for the Unique Constraint check for any POST request to LanguageSerializer for the code field. I know right now POST request isn't supported in LanguageSerializer. But inherently, this error validation was there, which goes away since I remove the validator from the serializer field. So this is kind of a re-instantiating of that validation.

Reply:
But you are right, I should add a check and raise ValidationError in case a language with that language code doesn't exist.

Reply:
Okay, I misread the not in the condition. But never mind the reverse validation should be there.